_config.yml

```diff
@@ -1,4 +1,6 @@
 name: OMERO User Help Website
+gems:
+  - jekyll-redirect-from
```

Comment:
Note `gems` is inserted here rather than at the bottom of the file to prevent conflicts between forks and live site.